bin/elasticity-flexture-L0/2_bending_of_lithosphere_under_line_loads.ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
<p>The image in this cell is not showing up (at least in ReviewNB)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Hmm, there is a typo again. It turns out that I messed up the files between the push here and the version on Hubzero :(</p><p> </p>